### ISSUE TITLE
[VAMC] System page side nav text does not scale with browser font size settings

### ISSUE THREAD

**Issue by qjac** [department-of-veterans-affairs/va.gov-cms#23990]
## Describe the defect
When Chrome browser settings are set to scale the font to 'very small' or 'very large', the text in the side navigation does not scale. It stays at 15px. There is likely code setting the absolute pixel value for this font size that needs to be updated to use relative sizing.

Additionally, when set to 'very large' the text in the 'In this section' (shown when it switches to the mobile layout) the text does scale, but overlaps the hamburger icon at some screen sizes.

## To Reproduce
1. Using Chrome, go to https://www.va.gov/boston-health-care/
2. Update font size setting to 'very small' at chrome://settings/appearance
3. Note that most text on the page shrinks,but the side nav text does not
4. Update font size setting to 'very large' at chrome://settings/appearance
3. Note that most text on the page enlarges, but the side nav text does not

## AC / Expected behavior
- [ ] Text in the sidebar nav shrinks when Chrome browser font size settings are set to 'very small'
- [ ] Text in the sidebar nav enlarges when Chrome browser font size settings are set to 'very large'
- [ ] 'In this section' text does not overlap the icon, even when browser font size settings enlarge the text

## Screenshots

<img width="951" height="760" alt="Image" src="https://github.com/user-attachments/assets/0a80ec13-382a-4ad9-8905-80a5b8f3f5c2" />
<img width="612" height="703" alt="Image" src="https://github.com/user-attachments/assets/47daacea-c6ea-4b1d-ab97-8b55323ab1a9" />

## Definition of done
- [ ] Development
   - [ ] Code completed 
   - [ ] PR write up complete
   - [ ] Automated tests passed 
   - [ ] Code review requested 
   - [ ] PR approved (meets established quality and security standards)
- [ ] Review instance created/available
  - [ ] Team members notified to start QA process
- [ ] QA Process
  - [ ] A11y review (meets accessibility standards)
  - [ ] Design Review (meets design & UX requirements)
    - [ ] Tested with different devices / screen sizes
  - [ ] Functional/feature review (meets acceptance criteria)
- [ ] QA feedback reviewed/addressed, if any
- [ ] Testing completed; dev team notified to deploy
- [ ] PO/PM/external verify in prod
- [ ] Documentation updated

**Comment by r-bartlett-gsa** [comment]
@AMT98 Tagging you on the ticket, as for some reason I still cannot assign you to some tickets.
